repo2docker/__main__.py

```diff
@@ -246,6 +254,13 @@ def make_r2d(argv=None):
     if args.appendix:
         r2d.appendix = args.appendix
 
+    for l in args.labels:
+        if "=" in l:
```

Comment:
FWIW, this is a perfect application of [str.partition](https://docs.python.org/3/library/stdtypes.html#str.partition):

```python
key, _, value = l.partition("=")
```